src/modules/getOuLevelAndGroupText.js

```diff
@@ -61,12 +65,26 @@ const getLevelAndGroupText = (items, metaData, isLevel) => {
         .join(', ')
 
     if (!staticOuNames) {
-        return `${allDynamicOuNames} ${
-            isLevel ? i18n.t('levels') : i18n.t('groups')
-        }`
+        if (isLevel) {
+            return i18n.t('{{allDynamicOuNames}} levels', {
+                allDynamicOuNames,
+            })
+        }
+
+        return i18n.t('{{allDynamicOuNames}} groups', {
+            allDynamicOuNames,
+        })
+    }
+
+    if (isLevel) {
+        return i18n.t('{{allDynamicOuNames}} levels in {{staticOuNames}}', {
+            allDynamicOuNames,
+            staticOuNames,
+        })
     }
 
-    return `${allDynamicOuNames} ${
-        isLevel ? i18n.t('levels in') : i18n.t('groups in')
-    } ${staticOuNames}`
+    return i18n.t('{{allDynamicOuNames}} groups in {{staticOuNames}}', {
```

Comment:
Not sure if we have a best practice for this, but generally I try to avoid multiple return statements. Now there's 4, and they all return the same thing, so might be better to change to a `let` that's returned in the end for readability?